client/ayon_hiero/api/menu.py

```diff
@@ -98,13 +98,13 @@ def menu_install():
     creator_action = menu.addAction("Create...")
     creator_action.setIcon(QtGui.QIcon("icons:CopyRectangle.png"))
     creator_action.triggered.connect(
-        lambda: host_tools.show_creator(parent=main_window)
+        lambda: host_tools.show_publisher(tab="create")
```

Comment:
Hiero is still a Qt UI right? If so, we should be parenting this still to the main window. E.g. like in [maya](https://github.com/ynput/ayon-maya/blob/5a8fb388522bc8d25ca03f29cf524bb69042fda0/client/ayon_maya/api/menu.py#L95) or like the older code we also pass the `parent=main_window` along here and with the other `show_publisher` call. Otherwise it's not parented to the main hiero UI and may pop 'behind' the interface accidentally.

Reply:
this was already addressed